tests/unit/core/TestInstallAddon.js

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import { renderIntoDocument } from 'react-addons-test-utils';
 
 import createStore from 'amo/store';
+import { setInstallState } from 'core/actions/installations';
```

Comment:
While I think I still prefer the action name first (I usually care more about which action I'm dealing with more), I see your point and maybe I'm just weird for wanting the verb after.

Okay cool, going forward I'll note to prefer `verb-subject(object)` order.